Routing/Generator/BcUrlGenerator.php

```diff
@@ -2,16 +2,8 @@
 
 namespace Hautelook\TemplatedUriRouter\Routing\Generator;
 
-use Symfony\Component\HttpKernel\Kernel;
 use Symfony\Component\Routing\Generator\CompiledUrlGenerator;
-use Symfony\Component\Routing\Generator\UrlGenerator;
 
-if (!class_exists('Symfony\Component\Routing\Generator\CompiledUrlGenerator')) {
-    abstract class BcUrlGenerator extends UrlGenerator
-    {
-    }
-} else {
-    abstract class BcUrlGenerator extends CompiledUrlGenerator
-    {
-    }
-}
+abstract class BcUrlGenerator extends CompiledUrlGenerator
```

Comment:
why not have `Rfc6570Generator` extend directly `CompiledUrlGenerator` and drop this class?

Reply:
Yep, makes sense. Done.